fenix/app/src/test/java/org/mozilla/fenix/shopping/ReviewQualityCheckFeatureTest.kt

```diff
@@ -4,6 +4,8 @@
 
 package org.mozilla.fenix.shopping
 
+import io.mockk.mockk
+import io.mockk.verify
```

Comment:
⚠️ We should use Fakes or real objects over mocks as much as possible. Mocks slow the tests, should be used as a last resort. If a test doesn't has mocks, let's be scpetical when adding a dependency, the same tests can be tested without using mocks.